### DISCUSSION TITLE
Schematics for 4.2.2 & 4.2.7 boards

### DISCUSSION THREAD

**Discussion by gothcha** [Jyers/Marlin#923] [General Discussion]
I have generated a schematic for the 4.2.7 board with TMC2225 drivers.

[Creality 4.2.7 - Schematic.pdf](https://github.com/Jyers/Marlin/files/6560155/Creality.4.2.7.-.Schematic.pdf)

I have also updated the 4.2.2 schematic to correctly show P3 pins being a system reset.

[Creality 4.2.2 - Schematic.pdf](https://github.com/Jyers/Marlin/files/6560156/Creality.4.2.2.-.Schematic.pdf)

**Comment by carlosaris** [comment]
I am sorry for my bad English, it's not my first language
Wow, the Creality 32 bit motherboard schematics!
Great job, thanks for sharing.
Reviewing the Marlin code of my Ender 3 Pro with motherboard 4.2.2, and comparing it with the schematic, it seems that PB2 should be connected to pin 5 of LCD Port.
Thanks again for sharing the schematic.

**Comment by gothcha** [comment]
@carlosaris Yes you are totally correct! I have checked the 4.2.7 board and indeed PB2 is connected to pin 5 of the LCD port and I can presume that this is also true of the 4.2.2 (which is in my printer) Thank you for finding this, I will correct both schematics.

**Reply by carlosaris** [reply]
Thanks @gothcha  for the updated schematics!

**Comment by gothcha** [comment]
Updated schematics, PB2 connection added.
[Creality 4.2.2 - Schematic 31-8-21.pdf](https://github.com/Jyers/Marlin/files/7083060/Creality.4.2.2.-.Schematic.31-8-21.pdf)
[Creality 4.2.7 - Schematic 31-8-21.pdf](https://github.com/Jyers/Marlin/files/7083061/Creality.4.2.7.-.Schematic.31-8-21.pdf)

**Comment by jetel73** [comment]
How to use the reset system P3 pins, just shorting them ?

**Comment by gothcha** [comment]
Yes this is correct

**Comment by jetel73** [comment]
Ok thx
Is the +5v powerful enough to power a 12v 0.4A fan without damaging the board ?
I'm using a Buck to power it continuously from the 24v output but it should be great to plug it directly on the board.

Edit : The SGM6232 that I assume is the converter that powers the USB is rated 2A in the datasheet ! It should work, am I right ?
https://file.remont-aud.net/baza/dc_dc/data/SGM6232.pdf
